src/components/Button/theme.ts

```diff
@@ -5,6 +5,8 @@ export const buttonTheme: FlowbiteButtonTheme = {
   base: 'group flex items-stretch items-center justify-center p-0.5 text-center font-medium relative focus:z-10 focus:outline-none transition-[color,background-color,border-color,text-decoration-color,fill,stroke,box-shadow]',
   fullSized: 'w-full',
   color: {
+    ghost:
```

Comment:
I believe that ghost should not be a color, instead it should be a flag like the `pill` or `outline`.